test/e2e/node/pre_stop.go

```diff
@@ -181,8 +179,8 @@ var _ = SIGDescribe("PreStop", func() {
 		testPreStop(f.ClientSet, f.Namespace.Name)
 	})
 
-	ginkgo.It("graceful pod terminated should wait until preStop hook completes the process [Flaky]", func() {
```

Comment:
this PR effectively change what the test is validating. Original one was testing that if gracePeriod allows it and the pod preStop hook is configured to be long enough, pod will be running that preStop hook for the whole duration. New test checks that pod will not exist after grace period. Which doesn't confirm that preStop hook was executed. Is it correct understanding or a proposed change?

Reply:
@SergeyKanzhelev yes, the test was trying to validate that after the gracePeriod, the pod still have 2 seconds of [extended grace period](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-termination) (which cause the flakyness). The proposed change is that even with a still running preStop hook, the pod eventually will be terminated.